### PR TITLE
imp: Snack example for Modal and Picker

### PR DESCRIPTION
**Pull request by Esemesek**
### Description

Yet another PR with snack examples. C:

### CODE REVIEW

**Comment by Esemesek** [comment]
@charpeni It works on Android and iOS simulators. It seems like there is no Modal implementation for web, so it is replaced with `div`.

**Comment by Esemesek** [comment]
I can add `Platform` check to this one, and on `web` render some kind of message.

**Comment by charpeni** [comment]
Hmm yeah. Related to #1219. This seems odd to me to default to web with those unsupported components 🤔 

Not sure what will be the best solution. cc @rachelnabors.

**Comment by Esemesek** [comment]
I think we are able to pass a parameter to our RemarkablePlugin. https://github.com/facebook/react-native-website/blob/master/website/core/RemarkablePlugins.js#L85 . I'll try to fix it tomorrow.

File: docs/picker.md [hunk 1]
```diff
@@ -5,16 +5,39 @@ title: Picker
...
+
+const styles = StyleSheet.create({
+  container: {
+    flex: 1,
+    justifyContent: 'center',
+    alignItems: 'center',
+  },
+  picker: {
+    width: 200,
+    height: 300,
```
Comment (ecreeth) [inline docs/picker.md]: Maybe **height: 30** :smile:

Comment (Esemesek) [reply docs/picker.md]: Thanks. It makes more sense for it to me 30 :P

**Comment by rachelnabors** [comment]
Folks, what's the status on this one? Looks like modal on web still isn't working.

**Comment by cpojer** [comment]
@Esemesek can you revert the change to Modal?

**Comment by cpojer** [comment]
Actually I just merged https://github.com/facebook/react-native-website/pull/1433 which adds the snack player to Picker, so let's just close this PR for now and take care of Modal later :)

**Pull request closed without merging**